string prior_pcd_str
string post_pcd_str
geometry_msgs/Point operated_position
---
bool is_success
